roles/apache/tasks/main.yml

```diff
@@ -1,77 +1,121 @@
 ---
 
+- include_vars:
+    dir: "{{ inventory_dir }}/private/vhosts"
+    name: vhost_specific_variables
+  tags: apache, virtual_hosts
+
+- name: Set vhosts
```

Comment:
could we move the apache stuff into an apache.yml and include it, would be nicer instead of having it in the main.yml so it can easier be extended with lets say nginx

Reply:
Something like this?

```
roles/
└── webserver/
    ├── apache.yml
    ├── main.yml
    └── nginx.yml
```

Reply:
yeah excactly

Reply:
Would you agree to make this change (which is a bit deep) in another PR? I'd like to reorganize on a stable basis.

Reply:
sure I currently try to find time to add something that works with exim and nginx to this role, so If i get around to it I will do that change anyway :D